src/test/java/io/vertx/ext/eventbus/bridge/tcp/TcpEventBusBridgeTest.java

```diff
@@ -319,6 +352,86 @@ public void testUnRegister(TestContext context) {
 
   }
 
+  @Test
+  public void testReplyFromClient(TestContext context) {
+    // Send a request from java and get a response from the client
+    NetClient client = vertx.createNetClient();
+    final Async async = context.async();
+    final String address = "test";
+    client.connect(7000, "localhost", conn -> {
+      context.assertFalse(conn.failed());
+
+      NetSocket socket = conn.result();
+
+      final FrameParser parser = new FrameParser(parse -> {
```

Comment:
use `context.assertAsyncSuccess` instead

Reply:
Done, and everywhere else it appeared in this file, I did the modification.